### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by NeoAnomaly** [GPUOpen-LibrariesAndSDKs/FireRays_SDK#56]
I found the getting started topic in the white paper. Can be there is a more complete version(tutorials, api reference, good practices, etc)?

**Comment by yozhijk** [comment]
Hi @NeoAnomaly,

Getting started in the white paper is a bit outdated. We are planning to release new version after the Siggraph. In the meantime you can use reference renderer to study best practices (at least I hope it is using best practices:))

**Comment by JaniKallankari** [comment]
Any progress with the documentation?

**Comment by yozhijk** [comment]
Hi @JaniKallankari ,

Still working on it, will hopefully have something next week.

**Comment by markusfehr** [comment]
Hi @yozhijk
I'm also looking forward for an up to date documentation.
Thanks :-)

**Comment by JaniKallankari** [comment]
Is there any plans to improve documentation in 2017?

**Comment by yozhijk** [comment]
@JaniKallankari Yes, I am currently doing significant changes in reference renderer in scene-interface branch. Will put the doc once I merge it in.
